topics/assembly/tutorials/assembly-quality-control/slides.html

```diff
@@ -215,57 +247,29 @@
 
 ### Assembly kmer content - Homozygous genomes
 
-<br>
-<br>
-
-K-mer spectra representation for Merqury (left) and KAT (rigth)
-
-.pull-left[
 .image-70[
 ![Merqury spectra-cn plot for S.cerevesiae S288C assembly](../../images/genome-qc/kmer-spectra-merqury-homozygous-good.png)
-]]
-
-.pull-right[
-.image-70[
-![KAT spectra-cn plot for S.cerevesiae S288C assembly](../../images/genome-qc/kmer-spectra-kat-homozygous-good.png)
-]]
+]
 
 {% icon congratulations %} Good kmer representation of reads in the assembly
 
 ---
 
 ### Assembly kmer content - Homozygous genomes
 
-</br>
-
-.pull-left[
 .image-70[
 ![Merqury spectra-cn plot for S.cerevesiae S288C assembly in case of missing contigs](../../images/genome-qc/kmer-spectra-merqury-homozygous-bad.png)
-]]
-
-.pull-right[
-.image-70[
-![KAT spectra-cn plot for S.cerevesiae S288C assembly in case of missing contigs](../../images/genome-qc/kmer-spectra-kat-homozygous-bad.png)
-]]
+]
 
 <i class="fa fa-thumbs-down"></i> Bad kmer representation of reads in the assembly
 
 ---
 
 ### Assembly kmer content - Heterozygous genomes
 
-</br>
-</br>
-
-.pull-left[
 .image-70[
 ![Merqury spectra-cn plot for highly heterozygous assembly and for which haplotig were purged or collapsed.](../../images/genome-qc/kmer-spectra-merqury-heterozygous-good.png)
-]]
-
-.pull-right[
-.image-70[
-![KAT spectra-cn plot for highly heterozygous assembly and for which haplotig were purged or collapsed.](../../images/genome-qc/kmer-spectra-kat-heterozygous-good.png)
```

Comment:
Is there a reason to remove all of the KAT images from the slide deck?

Reply:
Yes, the wrapper for KAT is still not available and Merqury offers more functions to evaluate the quality of assemblies even if the full range of possibilities is not presented (trio, QV, switch,etc).